Library/Homebrew/extend/ARGV.rb

```diff
@@ -268,6 +268,10 @@ def fetch_head?
     include? "--fetch-HEAD"
   end
 
+  def ignore_installed_specs?
+    include? "--ignore-installed-specs"
```

Comment:
> The fallback behaviour to stable seems like a very comfortable approach to me as well. 

This is what we do now.

> 1a godep depends on go
1b go is currently installed as devel
1c godep gets e.g. installed -s as a recommended option
2a asa go has a new version, that version is installed by regular upgrade process (right?)
2b godep is upgraded to the home-brew bottle build (would have to be implemented on your side I guess)

At `2b` brew upgrade `godep` should leave us with stable go and poured bottle.


> One thing that came to my mind, how are HEAD versions handled in this situation? Just the same as devel? maybe this is off topic but yeah...

HEADs have been already discussed in the outdated diff right after this one. HEAD stays HEAD.